setup.py

```diff
@@ -101,7 +101,7 @@ def recursive_data_files(directory, install_directory):
     "craft-cli",
     "craft-grammar",
     "craft-parts",
-    "craft-providers",
+    "craft-providers @ git+https://github.com/canonical/craft-providers@main#egg=craft-providers",
```

Comment:
craft-providers `1.23.1` is released, I don't think you need to bring this in

Reply:
great! I thought I had gotten rid of this!